test/simplifyExpression/simplify.test.js

```diff
@@ -46,7 +46,7 @@ describe('can simplify with division', function () {
     ['2x * 5x / 2', '5x^2'],
     ['2x * 4x / 5 * 10 + 3', '16x^2 + 3'],
     ['2x * 4x / 2 / 4', 'x^2'],
-    ['2x * y / z * 10', '20x * y / z'],
+    ['2x * y / z * 10', '(20x * y) / z'],
```

Comment:
This test case changed slightly because now this case is handled by `multiplyFractionSearch` ... think that's fine?